tasks/connectors/insight_appsec/insight_appsec_task.rb

```diff
@@ -0,0 +1,219 @@
+# frozen_string_literal: true
+
+require_relative "lib/insight_appsec_client"
+require "pry"
```

Comment:
Don't forget to remove the require "pry". I suspect you are using RubyMine, why don't you use RM debugger?

Reply:
shure) this habit came from another code editor 😅